### PR TITLE
Added submit/get/cancel-task methods

### PR DESCRIPTION
**Pull request by evgeniimv**
New stubbed methods "submitTesTask", "cancelTesTask" and "getTesTask" added in TesAdapterController and also in TesTaskService.
Also used auto-reformatted entities imports.

### CODE REVIEW

**Comment by evgeniimv** [comment]
I made fetch and merge with our current branch tes-support to up to date state. And also 
implemented three stubbed methods submit/get/cancel - TesTask.

File: tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesAdapterController.java
Comment (mzueva) [inline tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesAdapterController.java]: Please, do not use "star" imports. You can configure it for Idea using `Class count to use import with '*'` option.

File: tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesAdapterController.java
Comment (mzueva) [inline tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesAdapterController.java]: Value for `@PathVariable` annotation is not required, if name matches argument name.

Comment (mzueva) [reply tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesAdapterController.java]: This method also has one optional parameter: `view` (check the [spec](https://github.com/ga4gh/task-execution-schemas/blob/master/task_execution.swagger.json)). It is better to wait until #13  will be merged as it will provide enum for this parameter.

**Review by mzueva — CHANGES_REQUESTED** [review]

File: tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesAdapterController.java
Comment (evgeniimv) [reply tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesAdapterController.java]: fixed

File: tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesAdapterController.java
Comment (evgeniimv) [reply tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesAdapterController.java]: fixed

Comment (evgeniimv) [reply tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesAdapterController.java]: fixed

**Comment by evgeniimv** [comment]
fixed last issues. Added TaskView as enum parameter in method getTesTask().

**Review by mzueva — APPROVED** [review]

**Pull request merged**
